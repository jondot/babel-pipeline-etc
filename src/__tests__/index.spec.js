import { map, filter, reject, isNil } from 'ramda'

describe('babel operators', () => {
    it('should work', () => {
        expect(
            [1, 2, 3]
            |> map(_ => _ * 2)
            |> filter(_ => _ > 4)
            |> map(_ => _ * 4)).toMatchSnapshot()
    })
})