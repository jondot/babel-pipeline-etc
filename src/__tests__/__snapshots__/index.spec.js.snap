// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel operators should work 1`] = `
Array [
  24,
]
`;
